src/commands/synthetics/utils.ts

```diff
@@ -778,8 +778,8 @@ export const renderResults = ({
   return hasSucceeded ? 0 : 1
 }
 
-export const getDatadogHost = (useIntake = false, config: SyntheticsCIConfig) => {
-  const apiPath = 'api/v1'
+export const getDatadogHost = (useIntake = false, isUnstableApi = false, config: SyntheticsCIConfig) => {
```

Comment:
Arguably I think it would be easier to reason about if this function was taking an object so we have named fields. (`{useIntake: boolean, apiVersion: 'v1' | 'unstable', config: SyntheticsCIConfig}`)
If it has too deep implications, feel free to create a card in SW board